### ISSUE TITLE
Unable to scroll horizontally to the left when characters in code block exceed screen width

### ISSUE THREAD

**Issue by saurabh-sm** [willfaught/paige#62]
If there's a file with the following content:

```md
---
...
...
front: matter
...
paige: headers
...
...
---

## Some Heading

Some text

```json
{
  "examples": [
    {
      "description": "Placeholder text for description. A really long description that spans the entire width of the screen, such that scrolling to the left is only possible to a limited extent. The leftmost part get cut off.",
      "numbersOne": [1, 3],
      "numbersTwo": [2],
      "output": 2.00
    },
  ]
}
```

the left-most part of the text gets cut off and is not visible.

I write short descriptions when I highlight text as JSON, but sometimes I'm unable to avoid the width of text in the description. Since this is JSON, I can't split strings over multiple lines.

---

Is there a setting that I can declare in `hugo.toml` to make horizontal scrolling work ?

**Comment by willfaught** [comment]
Thanks for reporting. This is a bug. It should be fixed in v0.76.2. Can you confirm?

**Comment by saurabh-sm** [comment]
This works as well ! Thank you.
